server/segment_rpc_test.go

```diff
@@ -469,6 +477,261 @@ func TestServeSegment_InsufficientBalanceError(t *testing.T) {
 	assert.Equal("Internal Server Error", strings.TrimSpace(string(body)))
 }
 
+func TestServeSegment_DebitFees_SingleRendition(t *testing.T) {
+	orch := &mockOrchestrator{}
+	handler := serveSegmentHandler(orch)
+
+	require := require.New(t)
+
+	orch.On("VerifySig", mock.Anything, mock.Anything, mock.Anything).Return(true)
+
+	s := &BroadcastSession{
+		Broadcaster: stubBroadcaster2(),
+		ManifestID:  core.RandomManifestID(),
+		Profiles: []ffmpeg.VideoProfile{
+			ffmpeg.P720p60fps16x9,
+		},
+	}
+	seg := &stream.HLSSegment{Data: []byte("foo")}
+	creds, err := genSegCreds(s, seg)
+	require.Nil(err)
+
+	md, err := verifySegCreds(orch, creds, ethcommon.Address{})
+	require.Nil(err)
+
+	orch.On("ProcessPayment", net.Payment{}, s.ManifestID).Return(nil)
+	orch.On("SufficientBalance", s.ManifestID).Return(true)
+
+	tData := &core.TranscodeData{
+		Data:   []byte("foo"),
+		Pixels: int64(110592000),
+	}
+	tRes := &core.TranscodeResult{
+		TranscodeData: []*core.TranscodeData{tData},
+		Sig:           []byte("foo"),
+		OS:            drivers.NewMemoryDriver(nil).NewSession(""),
+	}
+	orch.On("TranscodeSeg", md, seg).Return(tRes, nil)
+	orch.On("DebitFees", md.ManifestID, mock.Anything, mock.Anything)
```

Comment:
You're right and I tried that, but it was not working and I didn't know why. 

Now I do,  this is because of #980 not being implemented.

```
--- Expected
+++ Actual
@@ -1,2 +1,2 @@
-(*net.PriceInfo)(pricePerUnit:2 pixelsPerUnit:3 )
+(*net.PriceInfo)(<nil>)
```

So we'll have to merge that and then I can rebase this PR of off master and implement actual values. 

Or can I rebase from that remote branch and not necessarily master?

Reply:
Either should be fine